assignments/callbacks.js

```diff
@@ -23,30 +23,67 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 
 function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
 
+getLength(items, function(x){
+  console.log(x);
+});
```

Comment:
You reuse this callback throughout; it'd be easier to assign it to a variable and also reduce repetition:
```javascript
// ...
const log = function(x){
  console.log(x);
}

getLength(items, log);
last(items, log);
// ...
```

Reply:
The other thing to note here is that `log` **is** a function. Specifically, it's a method on the `console` object. You could instead pass it as your callback function since it's unnecessary to pass it via another function:
```javascript
// ...
getLength(items, console.log);
last(items, console.log);
// ...
```